spec.md

```diff
@@ -213,6 +213,9 @@ const address = await flatDirectory.deploy();
 **Parameters**
 - `key` (string): The key of the data.
 - `data` (Buffer): The data to be uploaded.
+- `gasIncPct` (number, optional): The parameter is used to specify the percentage increase on the current default gas. 
```

Comment:
Did you intentionally omit this parameter in the README.md?

Reply:
Yes, because it should not be used in normal state.